pupil-spa/src/app/qr-code-arrival/qr-code-arrival.component.html

```diff
@@ -0,0 +1,14 @@
+<main id="content" class="js-content" role="main">
+        <div class="grid-row">
+            <header>
+                <h1 class="heading-xlarge">Redirecting</h1>
+            </header>
+            <div class="notice">
+                <strong class="visually-hidden">Please wait</strong>
+                <strong class="aa-copy-size">
+                    Redirecting to home page
```

Comment:
It would be good to have a link to click here, in case the redirection failed.

Reply:
good call.  this page never shows, but just in case...